### PR TITLE
check pem.Encode's return err before return

### PR DESCRIPTION
**Pull request by abel123**
**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rohitsardesai83** [comment]
@abel123, thanks for your contribution. Seems like there are conflicts. Could you please resolve the conflicts and update the PR again?

**Comment by abel123** [comment]
@rohitsardesai83 done

File: edge/pkg/common/util/config.go [hunk 1]
```diff
@@ -146,9 +146,8 @@ func createPEMfile(path string, pemBlock pem.Block) error {
 	if err != nil {
 		return err
 	}
-	if err := pem.Encode(file, &pemBlock); err != nil {
-		return err
```
Comment (abel123) [inline edge/pkg/common/util/config.go]: file will not be closed if return here. fixed in this pr

**Comment by m1093782566** [comment]
Nice catch!

LGTM, Thanks!

**Pull request merged**